.github/workflows/generated-ios-12-5-1-arm64-full-jit.yml

```
@@ -28,7 +28,7 @@ jobs:
   build:
     # NOTE: These builds will not run successfully without running on `pytorch/pytorch` due to the limitations
     #       of accessing secrets from forked pull requests and IOS' dependency on secrets for their build/test
-    if: ${{ github.event_name == 'push' || github.event.pull_request.head.repo.full_name == github.repository }}
+    if: ${{ github.event_name == 'push' || github.event.pull_request.head.repo.full_name == github.repository || github.repository_owner == 'pytorch' }}
```

```suggestion
    if: ${{ github.event_name == 'push' || (github.event.pull_request.head.repo.full_name == github.repository && github.repository_owner == 'pytorch') }}
```
Should be an and, no?

---

yeah that's my fault , i commit your suggeation